frontend/src/app/gaming/dropin/dropin.component.scss

```diff
@@ -0,0 +1,3 @@
+.dropin {
+    height: 400px;
```

Comment:
I've been using rems everywhere. This would be 25rem.

Reply:
cool good idea like that better